aubreylib/resource.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, identifier, metadataLocations, staticFileLocations,
         # Get transcriptions data
         self.transcriptions = get_transcriptions_data(
             meta_id=self.meta_id,
-            resource_type=self.desc_MD['resourceType'][0]['content'],
+            resource_type=self.desc_MD.get('resourceType', [{}])[0].get('content'),
```

Comment:
Should we add a test for there being no `resourceType`?

Reply:
I suppose....